asv_bench/benchmarks/algorithms.py

```diff
@@ -112,5 +119,25 @@ def time_series_timedeltas(self, df):
     def time_series_dates(self, df):
         hashing.hash_pandas_object(df['dates'])
 
+    def time_series_multiindexes(self, df):
```

Comment:
Hmm do we need to test single index vs mi?

Reply:
Removed the MultiIndex related changes